desktopfile-icon.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/qt/res/app.comaps.desktop.desktop b/qt/res/app.comaps.desktop.desktop
+index e11e4287..956fb2ff 100644
+--- a/qt/res/app.comaps.desktop.desktop
++++ b/qt/res/app.comaps.desktop.desktop
+@@ -7,7 +7,7 @@ Comment[ast]=Mapes detallaos del mundu ensin conexón
+ Comment[ca]=Mapes detallats del món sense connexió
+ Comment[es]=Mapas detallados del mundo sin conexión
+ Comment[ru]=Подробная оффлайновая карта мира
+-Icon=comaps
++Icon=app.comaps.desktop
```

Comment:
you change that with the command `desktop-file-edit`. but then are then even installed with that name by cmake?